src/DebugEngineHost/HostNatvisProject.cs

```diff
@@ -73,6 +81,84 @@ internal enum VSENUMPROJFLAGS
                 //  file as well as all virtual projects including nested (a.k.a. sub) projects
             };
 
+            /// <summary>
+            /// Gets the WorkspaceService from Microsoft.VisualStudio.Workspace
+            /// </summary>
+            /// <remarks>This package won't be automatically loaded by MEF, so we need to manually acquire exported MEF Parts.</remarks>
+            private static Lazy<IVsFolderWorkspaceService> WorkspaceService
+            {
+                get
+                {
+                    Lazy<IVsFolderWorkspaceService> workspaceService = null;
```

Comment:
Is this going to be lazy if the backing field is within the get property? Or am i not understanding the pattern? It seems like with this getter, every time WorkspaceService is accessed, it will call the serviceProvider and get the workplaceService again.